playwright-tests/tests/payments/create-payment-request.spec.js

```diff
@@ -58,13 +63,36 @@ test.describe("admin connected", function () {
   // }) => {
   // });
 
-  // test("cancel form should clear existing values", async ({
-  //   page,
-  //   instanceAccount,
-  //   daoAccount,
-  // }) => {
-  //   test.setTimeout(60_000);
-  // });
+  test("cancel form should clear existing values", async ({
+    page,
+    instanceAccount,
+    daoAccount,
+  }) => {
+    test.setTimeout(60_000);
+    await fillCreateForm(page, daoAccount, instanceAccount);
+    const cancelBtn = page
+      .locator(".offcanvas-body")
+      .getByRole("button", { name: "Cancel" });
+    await expect(cancelBtn).toBeAttached({ timeout: 10_000 });
+
+    // await cancelBtn.waitFor({ state: "attached", timeout: 10_000 });
+    // await cancelBtn.scrollIntoViewIfNeeded({ timeout: 10_000 });
+    cancelBtn.click();
+    await page.getByRole("button", { name: "Yes" }).click();
+
+    await clickCreatePaymentRequestButton(page);
+
+    await expect(await page.getByTestId("proposal-title").inputValue()).toBe(
+      ""
+    );
+    await expect(await page.getByTestId("proposal-summary").inputValue()).toBe(
+      ""
+    );
```

Comment:
can we check for all both proposal selection and non selection? or can you just add a test case for it with TODO.